python/ray/train/lightning/lightning_trainer.py

```diff
@@ -325,6 +336,36 @@ def __init__(
             resume_from_checkpoint=resume_from_checkpoint,
         )
 
+    @DeveloperAPI
+    def _create_air_checkpoint_config(
+        self, lightning_checkpoint_config: Optional[Dict] = None
+    ) -> CheckpointConfig:
+        """
+        Generate AIR CheckpointConfig based on provided Lightning checkpoint config.
+
+        PTL checkpointing logic:
+            no monitor + no save_top_k:  default = 1, only save last one
+            no monitor +    save_top_k:  if save_top_k != 1, save all
+               monitor + no save_top_k:  default = 1, save the best checkpoint
+               monitor +    save_top_k:  n/a
+        """
+
+        if not lightning_checkpoint_config:
+            lightning_checkpoint_config = {}
+
+        mode = lightning_checkpoint_config.get("mode", "min")
```

Comment:
Should we be setting `min` as a default? I feel like this should be required if they set a monitor.